docs/user/general-api.md

```diff
@@ -78,6 +78,25 @@ Library code should never call `Glean.initialize`, since it should be called exa
 Glean.initialize(applicationContext, Configuration(channel = "beta"))
 ```
 
+> **Note**: When the Glean SDK is consumed through Android Components, it is required to configure an HTTP client to be used for upload.
```

Comment:
I was going to suggest linking to the `concept.fetch` docs, but I can't really find the relevant place.  [This link](https://mozac.org/api/mozilla.components.concept.fetch/) is too low-level for what someone would need to know here.  So probably fine without the link, but I wish there was one...